.github/workflows/dest_meilisearch.yml

```diff
@@ -37,11 +37,10 @@ jobs:
         ports:
         - 7700:7700
         options: >-
-                 --health-cmd "wget --no-verbose --tries=1 --spider http://localhost:7700/health || exit 1"
```

Comment:
Maybe GitHub actions updated their `wget` version. When I run the old check locally on a health instance I get:
```
http://localhost:7700/health:
Remote file does not exist -- broken link!!!
```